changes/1747-noelevans.md

```diff
@@ -0,0 +1 @@
+Fix for issue 1747, properly representing Literal Enums when use\_enum\_values is True.
```

Comment:
```suggestion
Properly represent `Literal` Enums when `use_enum_values` is True.
```
The issue will be linked automatically thanks to the file name and history generation